legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -761,15 +773,17 @@ def resolution_json(resolution_revision) -> dict:
     @staticmethod
     def business_revision_json(business_revision, business_json):
         """Return the business revision as a json object."""
-        business_json["hasRestrictions"] = business_revision.restriction_ind
+        business_json["hasRestrictions"] = (
+            business_revision.restriction_ind if business_revision.is_legal_entity else False
+        )
         business_json["dissolutionDate"] = (
             LegislationDatetime.format_as_legislation_date(business_revision.dissolution_date)
             if business_revision.dissolution_date
             else None
         )
         business_json["restorationExpiryDate"] = (
             LegislationDatetime.format_as_legislation_date(business_revision.restoration_expiry_date)
-            if business_revision.restoration_expiry_date
+            if business_revision.is_legal_entity and business_revision.restoration_expiry_date
             else None
```

Comment:
The json attributes specific to legal entity could all be assigned under a single `if` statement (instead of using multiple `business_revision.is_legal_entity` checks) if we are okay with these attributes not appearing at all in the alternate name json object.

Reply:
yeah, i think they can all go under a single `if` statement.